apps/web/components/ui/tambo/thread-content.tsx

```diff
@@ -154,6 +154,8 @@ const ThreadContentMessages = React.forwardRef<
               className={cn(
                 "flex w-full",
                 message.role === "assistant" ? "justify-start" : "justify-end",
+                // messages with parent should be slightly dimmed out
```

Comment:
this is a temporary fix that is just for us internally - we'll do a proper UI for this next